src/util/PolarUtils.ts

```diff
@@ -141,15 +141,17 @@ const reverseFormatAngleOfSetor = (angle: number, { startAngle, endAngle }: Geom
   return angle + min * 360;
 };
 
-export const inRangeOfSector = ({ x, y }: Coordinate, sector: GeometrySector) => {
+export const inRangeOfSector = ({ x, y }: Coordinate, sector: GeometrySector): RangeObj | null => {
   const { radius, angle } = getAngleOfPoint({ x, y }, sector);
   const { innerRadius, outerRadius } = sector;
 
   if (radius < innerRadius || radius > outerRadius) {
+    // @ts-expect-error usages of this method expect it to always return RangeObj, not boolean
```

Comment:
hmm, it looks like the code kind of handles the true/false cases no? All good

Reply:
Either it handles or we have not yet discovered the combination of props and data that breaks.

Reply:
got it. I tried to follow it a little bit and it _looks_ like it does? But allowing null to return should cover the falsy case and returning only true seems incorrect